transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannelConfig.java

```diff
@@ -371,4 +377,31 @@ public EpollDatagramChannelConfig setReusePort(boolean reusePort) {
             throw new ChannelException(e);
         }
     }
+
+    /**
+     * Returns {@code true} if <a href="http://man7.org/linux/man-pages/man7/ip.7.html">IP_TRANSPARENT</a> is enabled,
+     * {@code false} otherwise.
+     */
+    public boolean isIpTransparent() {
+        try {
+            return datagramChannel.socket.isIpTransparent();
+        } catch (IOException e) {
+            throw new ChannelException(e);
+        }
+    }
+
+    /**
+     * If {@code true} is used <a href="http://man7.org/linux/man-pages/man7/ip.7.html">IP_TRANSPARENT</a> is enabled,
+     * {@code false} for disable it. Default is disabled.
+     */
+    public EpollDatagramChannelConfig setIpTransparent(boolean ipTransparent) {
+        try {
+            datagramChannel.socket.setIpTransparent(ipTransparent);
+            return this;
+        } catch (IOException e) {
+            throw new ChannelException(e);
+        }
+    }
+
 }
+
```

Comment:
revert

Reply:
revert?

Reply:
Yes please revert this change ...

Reply:
Do you mean the entire PR? Why?

Reply:
No just the one line change above the comment